### PR TITLE
[kesch] Python 3.6.2 extensions for MCH

### PR DESCRIPTION
**Pull request by kraushm**
Extra packages for Python 3.6.2 requested by Meteo Swiss.

### CODE REVIEW

**Comment by kraushm** [comment]
Jenkins did not show any problem. Should be good to merge.

**Comment by lucamar** [comment]
retest this please

**Comment by lucamar** [comment]
retest this please

File: easybuild/easyconfigs/b/bokeh/bokeh-0.12.3-gmvolf-17.02-Python-3.6.2.eb [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+]
+
+exts_list = [
+    ('Jinja2', '2.8', {
+        'source_urls': ['https://pypi.python.org/packages/source/J/Jinja2'],
+    }),
+    ('requests', '2.11.1', {
+        'source_urls': ['https://pypi.python.org/packages/source/r/requests'],
+    }),
+    ('tornado', '4.4.2', {
```
Comment (victorusu) [inline easybuild/easyconfigs/b/bokeh/bokeh-0.12.3-gmvolf-17.02-Python-3.6.2.eb]: Why not use tornado 4.5.1 like in the IPython?

File: easybuild/easyconfigs/b/bokeh/bokeh-0.12.3-gmvolf-17.02-Python-3.6.2.eb [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+dependencies = [
+    ('Python', '3.6.2'),
+    ('PyYAML', '3.12', versionsuffix),
+]
+
+exts_list = [
+    ('Jinja2', '2.8', {
+        'source_urls': ['https://pypi.python.org/packages/source/J/Jinja2'],
+    }),
+    ('requests', '2.11.1', {
```
Comment (victorusu) [inline easybuild/easyconfigs/b/bokeh/bokeh-0.12.3-gmvolf-17.02-Python-3.6.2.eb]: Why not use 2.13.0 like in the IPython?

**Comment by kraushm** [comment]
retest this please

**Pull request merged**